app/dashboard/views/custom_alias.py

```diff
@@ -281,14 +297,6 @@ def verify_prefix_suffix(user: User, alias_prefix, alias_suffix) -> bool:
             LOG.exception("User %s submits a wrong alias suffix %s", user, alias_suffix)
             return False
 
-        random_word_part = alias_domain_prefix[1:]
```

Comment:
Let's wait until the Safari extension (cf my comment on the Discussion thread) getting updated and then we can merge this PR :)

Reply:
@developStorm For the heads-up, the new Safari version has been released on Monday so I think we can confidently merge that change on next Monday :).

Reply:
Are we ready to merge now? 👀

Reply:
Not yet 🙁 , I still see the old endpoint being called by Safari extension so I guess the new version hasn't reached all devices yet. Let's wait for another week to be totally sure.